src/navigation/index.js

```diff
@@ -1,18 +1,46 @@
 import React from 'react';
 import {NavigationContainer} from '@react-navigation/native';
 import {createStackNavigator} from '@react-navigation/stack';
-import LandingPage from '../screens/LandingPage'
+import LandingPage from '../screens/LandingPage';
+import Modals from '../screens/Modals';
 
 const Stack = createStackNavigator();
 
 function AppNavigator() {
   return (
     <NavigationContainer>
-      <Stack.Navigator headerMode="none">
+      <Stack.Navigator
+        headerMode="none"
+        screenOptions={{
+          headerShown: false,
+          cardStyle: {backgroundColor: 'transparent'},
+          cardOverlayEnabled: true,
+          cardStyleInterpolator: ({current: {progress}}) => ({
+            cardStyle: {
+              opacity: progress.interpolate({
+                inputRange: [0, 0.5, 0.9, 1],
+                outputRange: [0, 0.25, 0.7, 1],
+              }),
+            },
+            overlayStyle: {
+              opacity: progress.interpolate({
+                inputRange: [0, 1],
+                outputRange: [0, 0.5],
+                extrapolate: 'clamp',
+              }),
+            },
+          }),
+        }}
+        mode="modal">
         <Stack.Screen name="LandingPage" component={LandingPage} />
+        <Stack.Screen
+          name="Modals"
```

Comment:
https://www.bigbinary.com/learn-react-native/navigation-best-practices#use-constants-for-screen-names